### PR TITLE
Latexified and Jaxified the zipfs_law_plot notebook

### PR DESCRIPTION
**Pull request by Madhav-Kanda**
## Description

Latexified and Jaxified the code of the [notebook of zipfs_law_plot](https://github.com/probml/pyprobml/blob/master/notebooks/book2/02/zipfs_law_plot.ipynb)
### Figure Number
Book 2- Figure : 2.4

### Figures

[gist](https://gist.github.com/Madhav-Kanda/b25d64db6c28c892b847451c35c56bc8)
### Issue 

#42 

## Steps

- [x] Step1 - latexified the code
- [x] Step2 - jaxified the code

## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

## Potential problems/Important remarks

The code generates two plots although only one plot has been shown in the Book-2. Kindly update whether the second figure should be kept in the code or not.

### CODE REVIEW

File: notebooks/book2/02/zipfs_law_plot.ipynb [hunk 1]
```diff
@@ -0,0 +1,311 @@
+{
```
Comment (karm-patel) [inline notebooks/book2/02/zipfs_law_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.        %pip install git+https://github.com/probml/probml-utils.git
</pre><p>put <code>-qq</code> flag </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (karm-patel) [inline notebooks/book2/02/zipfs_law_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    latexify(width_scale_factor=1.70, fig_height=2.25)
</pre><p>set <code>fig_height=2</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (karm-patel) [inline notebooks/book2/02/zipfs_law_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    N = jnp.sum(wordcounts)
</pre><p>what is c, x, N, a here? please use meaning full variable names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (Madhav-Kanda) [reply notebooks/book2/02/zipfs_law_plot.ipynb:1]: '**a**' and '**x**' are the variables as shown in the book for the Zipf law equation so could you please suggest to me a different name for them. Also, c is the rank from which the Zipf law plotting starts and  N is the total number of words in the passage so is the name: "**start_rank**" and " **total_words**" fine?

Comment (karm-patel) [reply notebooks/book2/02/zipfs_law_plot.ipynb:1]: Yes, you can put the variable names by your own.

**Comment by Madhav-Kanda** [comment]
@karm-patel Bhaiya, I have made the changes as stated by you, could you please check once and confirm.

**Comment by karm-patel** [comment]
Yes, you should keep code of all figures, but just latexify only needed figures.
1. Use `sns.despine()` to remove right and top spines

After this you can create a PR in main repo

**Comment by Madhav-Kanda** [comment]
@karm-patel bhaiya, I have removed the top and right spines could you please check once before I create a PR on the main repo.

**Comment by karm-patel** [comment]
Paste updated figure here

**Comment by Madhav-Kanda** [comment]
image

**Comment by karm-patel** [comment]
Looks good to me, go ahead

**Pull request merged**